src/cluster/src/communication.rs

```diff
@@ -0,0 +1,341 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+//! Code to spin up communication mesh for a compute replica.
+//!
+//! The startup protocol is as follows:
+//! The compute controller in `environmentd`, after having connected to all the
+//! `clusterd` processes in a replica, sends each of them a `CreateTimely` command
```

Comment:
It would be helpful if you made the delete of this file from compute part of the same commit, so it can be verified that it's not changing.

Reply:
Ill see what I can do; im not a git expert

Reply:
wowww just removing it in the commit below, git figures outs its a move+edit, thats amazing